gateway/cli/src/main.rs

```diff
@@ -77,7 +78,9 @@ pub enum Commands {
 }
 
 #[tokio::main]
-async fn main() {
+async fn main() -> anyhow::Result<()> {
+    TracingSetup::default().init()?;
```

Comment:
@jkitman for the gateway, we probably want tracing on the following key binaries:
- [gatewayd](https://github.com/fedimint/fedimint/blob/master/gateway/ln-gateway/src/bin/gatewayd.rs)
- [cln_extension](https://github.com/fedimint/fedimint/blob/master/gateway/ln-gateway/src/bin/cln_extension.rs)

The following binaries don't seem that important / are about to be phased out
- [ln_gateway](https://github.com/fedimint/fedimint/blob/master/gateway/ln-gateway/src/bin/ln_gateway.rs) - to be phased out. superseded by `gatewayd`
- [cli](https://github.com/fedimint/fedimint/blob/master/gateway/cli/src/main.rs) - current file. Just a cli tool

Reply:
Can't do plugins, have to use the env var, but added to gatewayd.

Reply:
ack. `cln_extension` plugs in to CLNs logging system